packages/styles/package.json

```diff
@@ -33,17 +33,17 @@
     "schematics:build": "copyfiles schematics/migrations.json dist && tsc -p tsconfig.schematics.json"
   },
   "dependencies": {
-    "@angular-devkit/core": "14.2.10",
-    "@angular-devkit/schematics": "14.2.10",
-    "@angular/core": "14.2.12",
+    "@angular-devkit/core": "<=15.0.4",
+    "@angular-devkit/schematics": "<=15.0.4",
+    "@angular/core": "<=15.0.4",
```

Comment:
Does this prevent renovate from sending updates?

Reply:
We will see next monday ;)
I used syntax, which is explained here: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies